### ISSUE TITLE
Missing argument 1 for Ps_EmailGenerator::getTemplatesToBuild()

### ISSUE THREAD

**Issue by ludohenin** [PrestaShop/ps_emailgenerator#5]
I'm getting this error from the BO when trying to configure it:

```
Warning: Missing argument 1 for Ps_EmailGenerator::getTemplatesToBuild(), called in 
/var/data/jds_shop/dev/modules/ps_emailgenerator/controllers/admin/AdminEmailGeneratorController.
php on line 42 and defined
```

**Comment by ludohenin** [comment]
`Locales` should actually be automatically set from the enabled languages.

**Comment by Quetzacoalt91** [comment]
Fixed in #16
